<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>todoItem</class>
 <widget class="QWidget" name="todoItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="timeText">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4/25/24 20：00</string>
   </property>
  </widget>
  <widget class="QLabel" name="msgText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>申请添加您为好友</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disagreeButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
  <widget class="QPushButton" name="agreeButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>同意</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>friendName</string>
   </property>
  </widget>
  <widget class="QPushButton" name="profileButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
